src/server/src/entities/studentToJob.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
+import { Student } from '../entities/student';
+import { Job } from '../entities/job';
+
+@Entity()
+export class StudentToJob {
```

Comment:
I think a better name for this entity would be JobApplication

Reply:
Done.